src/index.js

```diff
@@ -454,6 +459,8 @@ export default class Bili extends EventEmitter {
               preamble: banner
             }
           }),
+        options.pretty &&
+          prettierPlugin(getPrettierConfig()),
```

Comment:
what about using `this.localRequire` so that we don't need to include additional rollup plugin.

Reply:
umm yeah great, I will change this one :+1: